demo/react-native/App.tsx

```diff
@@ -212,33 +249,52 @@ export default class App extends Component<Props, State> {
     }
   }
 
+  async showContextInfo() {
+    if (!this.state.contextInfo) {
+      await this._startPicovoice();
+      this._stopProcessing();
+    }
+    if (!this.state.isError) {
+      this.setState({ showContextInfo: true });
+    }
+  }
+
   render() {
     return (
       <View style={[styles.container]}>
         <View style={styles.statusBar}>
           <Text style={styles.statusBarText}>Picovoice</Text>
+          <View style={styles.statusBarButtonContainer}>
+            <TouchableOpacity
+              style={{ backgroundColor: '#ffd105' }}
+              onPress={() => this.showContextInfo()}>
+              <Text style={styles.statusBarButtonStyle}>Context Info</Text>
+            </TouchableOpacity>
+          </View>
         </View>
 
-        <View
-          style={{
-            flex: 0.35,
-            justifyContent: 'center',
-            alignContent: 'center',
-          }}>
-          <TouchableOpacity
-            style={{
-              width: '50%',
-              height: '50%',
-              alignSelf: 'center',
-              justifyContent: 'center',
-              backgroundColor: this.state.isError ? '#cccccc' : '#377DFF',
-              borderRadius: 100,
-            }}
-            onPress={() => this._toggleListening()}
-            disabled={this.state.buttonDisabled || this.state.isError}>
-            <Text style={styles.buttonText}>{this.state.buttonText}</Text>
-          </TouchableOpacity>
-        </View>
+        <Modal
+          animationType="slide"
+          transparent={true}
+          visible={this.state.showContextInfo}>
+          <View style={styles.modalView}>
+            <ScrollView style={{ flex: 0.95, marginBottom: 10 }}>
+              <Text>{this._picovoiceManager?.contextInfo}</Text>
```

Comment:
I think you meant to use the `contextInfo` from your state. Using this I can't see the context info unless the app is in listening mode.